<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuInstruments">
    <property name="title">
     <string>Instruments</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionPencil"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionCircle"/>
    <addaction name="separator"/>
    <addaction name="actionchooseColor"/>
    <addaction name="actionfilling"/>
    <addaction name="actionUndo_2"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAdout"/>
   </widget>
   <addaction name="menuInfo"/>
   <addaction name="menuFile"/>
   <addaction name="menuInstruments"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionchooseColor"/>
   <addaction name="actionfilling"/>
   <addaction name="separator"/>
   <addaction name="actionPencil"/>
   <addaction name="actionLine"/>
   <addaction name="actionCircle"/>
   <addaction name="actionRectangle"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New </string>
   </property>
   <property name="toolTip">
    <string>Create new image</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open exsisted image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save image as .png</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/new/img/line.png</normaloff>:/new/img/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Line</string>
   </property>
  </action>
  <action name="actionPencil">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/new/img/pencil.png</normaloff>:/new/img/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Pencil</string>
   </property>
   <property name="toolTip">
    <string>Pencil</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/new/img/191781-200.png</normaloff>:/new/img/191781-200.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/new/img/circle.png</normaloff>:/new/img/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="toolTip">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionAdout">
   <property name="icon">
    <iconset>
     <normaloff>:/new/res/Img/about.png</normaloff>:/new/res/Img/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
  </action>
  <action name="actionchooseColor">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/new/img/color.png</normaloff>:/new/img/color.png</iconset>
   </property>
   <property name="text">
    <string>Choose color</string>
   </property>
   <property name="toolTip">
    <string>Choose a color </string>
   </property>
  </action>
  <action name="actionfilling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/new/img/filler.png</normaloff>:/new/img/filler.png</iconset>
   </property>
   <property name="text">
    <string>filling</string>
   </property>
   <property name="toolTip">
    <string>Fill area</string>
   </property>
  </action>
  <action name="actionUndo_2">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo action</string>
   </property>
  </action>
  <action name="actionactionPolygon">
   <property name="text">
    <string>actionPolygon</string>
   </property>
   <property name="toolTip">
    <string>actionPolygon</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
